configs/components/curl.rb

```diff
@@ -4,18 +4,45 @@
   pkg.url "http://buildsources.delivery.puppetlabs.net/curl-#{pkg.get_version}.tar.gz"
 
   pkg.build_requires "openssl"
+  pkg.build_requires "puppet-ca-bundle"
+
+  prefix = settings[:prefix]
+  pem_file = "#{settings[:prefix]}/ssl/cert.pem"
+
+  make = platform[:make]
+
+  if platform.is_windows?
+    pkg.build_requires "pl-zlib-#{platform.architecture}"
+    pkg.build_requires "runtime"
+
+    arch = platform.architecture == "x64" ? "64" : "32"
+    pkg.environment "PATH" => "#{settings[:gcc_bindir]}:$$PATH"
+    pkg.environment "CYGWIN" => settings[:cygwin]
+    pkg.environment "CC" => settings[:cc]
+    pkg.environment "CXX" => settings[:cxx]
```

Comment:
If you want to... I don't want to go back and rebuild all those packages just for this though